config/foreman.migrations/20240505103832_change_db_pool_size.rb

```diff
@@ -0,0 +1,4 @@
+if answers['foreman'].is_a?(Hash) && answers.dig('foreman', 'db_pool')
+  puma_threads_max = answers.dig('foreman', 'foreman_service_puma_threads_max').to_i
+  answers['foreman']['db_pool'] = [answers.dig('foreman', 'db_pool').to_i, puma_threads_max + 4].max
+end
```

Comment:
This would reset *any* `db_pool` setting, while we only want to reset those that have it at the default (`5`), right?
Also, I think it's sufficient to delete the value and let Kafo/Puppet recalculate it, that way you don't need to keep the calculation logic in multiple places

Reply:
If `foreman_service_puma_threads_max` is somehow `nil`, `puma_threads_max` is `0`. Then the value comes down to 0 + 4. Probably an edge case, but I'm a big fan of defensive code.